dataprofiler/profilers/profile_builder.py

```diff
@@ -1399,13 +1425,23 @@ def _remove_data_labelers(self):
         """
         data_labelers = {}
 
-        # Delete data labeler for profiler
-        data_labeler_options = self.options.structured_options.data_labeler
-        if data_labeler_options.is_enabled \
-                and data_labeler_options.data_labeler_object is not None:
-            data_labelers["data_labeler"] = data_labeler_options \
-                                            .data_labeler_object
-            data_labeler_options.data_labeler_object = None
+        # Delete data labeler from structured options
```

Comment:
instead fo delete, lets use the word remove like above. also, lets have the same format for both, either both determine if enabled then remove like this or determine if enabled and then remove for struct, then for unstruct.
this could potentially be a util function that both could use.